xknx.yaml

```diff
@@ -86,9 +86,9 @@ groups:
         Attic.Climate: {group_address_temperature: '1/7/7', group_address_operation_mode_protection: '1/7/8', group_address_operation_mode_night: '1/7/9', group_address_operation_mode_comfort: '1/7/10'}
         Cellar.Climate: {group_address_temperature: '1/7/11', group_address_controller_status: '1/7/12', group_address_controller_status_state: '1/7/13'}
 
-    # Cyclic sending of time to the KNX bus
-    time:
-        General.Time: {group_address: '2/1/2'}
+    # Cyclic sending of datetime to the KNX bus
```

Comment:
Date and Time are different datatypes in the knx world. Most sensors/actors have two groups for date and time, i never seen a combined datetime group in the real world.

Reply:
there are actually three types:
* Time (10.001, 3 octets)
* Date (11.001, 3 octets)
* DateTime (19.001, 8 octets)
With my change, you are able to choose which of those you need.
In real world, the [MDT Glastaster II Smart](http://www.mdt.de/Glastaster_Smart.html) is one of the KNX devices that needs a DateTime to work.